packages/zowe-explorer-api/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ All notable changes to the "zowe-explorer-api" extension will be documented in t
 - Added new `fetchAttributes` API to `IZoweUSSTreeNode` to fetch latest attributes for UNIX files. [#3238](https://github.com/zowe/zowe-explorer-vscode/issues/3238)
 - Added new `directConnectLogin` and `directConnectLogout` to the ZoweVsCodeExtension class. [#3346](https://github.com/zowe/zowe-explorer-vscode/issues/3346)
 - Added new optional `refreshFavorites` to IZoweTree interface. [#3470](https://github.com/zowe/zowe-explorer-vscode/issues/3470)
+- The `certFile` and `certKeyFile` are now supported fields for zosmf session/profile. [#2373](https://github.com/zowe/zowe-explorer-vscode/issues/2373)
```

Comment:
- We should say what `certFile` and `certKeyFile` are: "The `certFile` and `certKeyFile` **methods/profiles/whatever** are now supported fields ... "
- We should clarify what we mean by the slash: " ... for z/OSMF **sessions and profiles**."

Reply:
could we just copy the other changelog entry 
- Implemented Client Side Certificate authentication support for z/OSMF profiles. [#2373](https://github.com/zowe/zowe-explorer-vscode/issues/2373)

Reply:
we can polish the CHANGELOGs in the release PR for 3.2 so this is good to merge 😄